### PR TITLE
Add Glaes Package

### PR DESCRIPTION
**Pull request by julian-belina**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by julian-belina** [comment]
I accept to be maintainer.  @conda-forge/staged-recipes please review my recipe.

File: recipes/glaes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - numpy
+    - descartes
+    - pandas
+    - scipy
+    - matplotlib-base
+    - scikit-learn
+    - jupyter
+    - notebook
+    - jupyterlab
+test:
```
Comment (ocefpaf) [inline recipes/glaes/meta.yaml:34]:
```suggestion

test:
```

File: recipes/glaes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - pip
+    - pytest
+    - pytest-cov
+
+about:
+  home: https://github.com/FZJ-IEK3-VSA/glaes
+  license: MIT
+  license_file:
+    - LICENSE.txt
+  summary: GLAES is a framework for conducting land eligibility analyses and is designed to easily incorporate disparate geospatial information from a variety of sources into a unified solution.
```
Comment (ocefpaf) [inline recipes/glaes/meta.yaml:52]: This summary is quite long! Why not use the upstream one: `Geospatial Land Availability for Energy Systems`.

You can add a multi-line `description` if you want to keep that BTW.

File: recipes/glaes/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (ocefpaf) [inline recipes/glaes/LICENSE.txt:1]: This file is upstream no need to copy it here.

**Review by ocefpaf — CHANGES_REQUESTED** [review]

File: recipes/glaes/meta.yaml (hunk 2 above)
Comment (julian-belina) [reply recipes/glaes/meta.yaml:53]: Good suggestion. I was not sure about the appropiate length

File: recipes/glaes/LICENSE.txt (hunk 3 above)
Comment (julian-belina) [reply recipes/glaes/LICENSE.txt]: Thanks for the suggestion. I was not aware i can reference it from there

**Comment by ocefpaf** [comment]
The CIs did not start here. Let's close and re-open to toggle them.

**Pull request merged**